### PR TITLE
<Loader/> - added circle loader

### PR DESCRIPTION
**Pull request by wtfil**
loader

### CODE REVIEW

File: src/components/loader/loader.st.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
+:import {
+    -st-from: "../../style/project.st.css";
+    -st-named:
+		PR2,
+		BK2;
+}
+:vars {
+	size: 100px;
+	trackWidth: 5px;
```
Comment (tyv) [inline src/components/loader/loader.st.css]: this should be part of theme.

File: src/components/loader/loader.st.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  	height: 100%;
+  	overflow: hidden;
+  	box-sizing: border-box;
+}
+
+.track {
+	width: value(size);
+	height: value(size);
+  	box-sizing: border-box;
+  	border: 5px solid value(PR2);
```
Comment (tyv) [inline src/components/loader/loader.st.css]: `5px` should be part of theme

File: src/components/loader/loader.st.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+.right .track {
+	animation-delay: 1.2s;
+}
+
+.text {
+	display: block;
+	text-align: center;
+	color: value(BK2);
+	text-align: center;
+	margin-top: 1.5em;
```
Comment (tyv) [inline src/components/loader/loader.st.css]: this is should be part of theme

**Comment by wtfil** [comment]
@tyv updated

File: test/components/loader.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+                    expect(renderer.select('LOADER')).to.not.null;
+                });
+
+                it('should render children', () => {
+                    expect(renderer.select('TEST_DIV')).to.not.null;
+                });
+            });
+        });
+    });
+});
```
Comment (tyv) [inline test/components/loader.spec.tsx]: test description should be more abstract

File: src/components/loader/index.ts [hunk 5]
```diff
@@ -0,0 +1 @@
+export * from './circle-loader';
```
Comment (tyv) [inline src/components/loader/index.ts]: if Loader can accept children, it should be exported because it has sense actually

Comment (wtfil) [reply src/components/loader/index.ts]: agree

**Pull request merged**